### DISCUSSION TITLE
Changing a task to calendar or notes

### DISCUSSION THREAD

**Discussion by DickinsonSa** [TechbeeAT/jtxBoard#325] [Q&A] [answered]
Hi all, probably not the best title for this question, but this is my puzzle...

When I'm editing a task, I can change it to either calendar or notes from the top drop-down. However, the task remains in my tasks list - it doesn't move anywhere.

What's the point of having this functionality, what can I achieve by moving tasks (or for that matter calendar or notes) to any other category?

Am I missing out on a great feature here by not understanding how to use it better?

Thanks for your time.

**Comment by patrickunterwegs** [comment]
Hey @DickinsonSa , I am not quite sure which functionality you mean. 
Do you mean the export functionalities in the top me unto export entries as .ics?

Or do you mean the copy functionality at the bottom of the screen next to the trashcan?

**Reply by DickinsonSa** [reply]
Hi Patrick, none of those. It's the drop-down to the left of the colour picker.

It shows other groups in a collection: notes, calendar. 

So my question was: what's its purpose in life?

**Reply by patrickunterwegs** [reply, marked as answer]
So you mean the collections selector?

If you have multiple remote collections (e.g. synchronised through DAVx5) you can have multiple collections/calendars in the selection. If you put an entry in a wrong one, you can just easily move it to another one. This can be especially relevant when you have a collection shared with other people and a personal one.

**Reply by DickinsonSa** [reply]
Aaah, gottit.
Many thanks.
